views/extras.tmpl

```diff
@@ -165,6 +165,59 @@
             console.log("ping. users online: ", evt)
             ws.send("pong", "copy")
         });
+        var updateBlockData = function (event) {
+            console.log("Received newblock message", event);
+            var newBlock = JSON.parse(event);
+            var confirms = $(".confirmations")
+            $.each(confirms, function(i, v) {
```

Comment:
There's an edge case here where if a socket update is missed, the number of confirmations will get out of sync with the backend.

Maybe we could set the block height of when something was mined at as a data attribute eg `$('[data-confirmation-block-height]')` on the span that holds the confirmation value. 
Then we can derive confirmations by subtracting b.height from that value whenever new block comes in.